client/src/components/desk/DeskContainer.tsx

```diff
@@ -80,10 +103,10 @@ function DeskContainer() {
         />;
     }
     const requests = query.data.requests;
-    const submitted = getRequestsWithStatus(requests, SUBMITTED);
-    const approved = getConsolidatedRequestsWithStatus(requests, APPROVED);
-    const readyForPickup = getConsolidatedRequestsWithStatus(requests, READY_FOR_PICKUP);
-
+    const submitted = getRequestsWithStatus(requests, [SUBMITTED]);
+    const approved = getConsolidatedRequestsWithStatus(requests, [APPROVED]);
+    const readyForPickup = getConsolidatedRequestsWithStatus(requests, [READY_FOR_PICKUP]);
+    const readyForReturn = getConsolidatedRequestsWithStatus(requests, [FULFILLED, LOST, DAMAGED]);
```

Comment:
how would lost or damaged items be marked for return? aren't they states that are transitioned to after return?

Reply:
Lost/damaged items stay in the Returns list